pkg/monitor/monitor/monitor_manager.go

```diff
@@ -427,3 +435,64 @@ func (m *MonitorManager) patchTidbClusterStatus(tcRef *v1alpha1.TidbClusterRef,
 	_, err = m.deps.Clientset.PingcapV1alpha1().TidbClusters(tc.Namespace).Patch(tc.Name, types.MergePatchType, mergePatch)
 	return err
 }
+
+func (m *MonitorManager) smoothMigrationToStatefulSet(monitor *v1alpha1.TidbMonitor) error {
+	exist, err := m.deps.TypedControl.Exist(client.ObjectKey{
+		Namespace: monitor.Namespace,
+		Name:      monitor.Name,
+	}, &appsv1.Deployment{})
+	if err != nil {
```

Comment:
Is this reasonable for the new creation case?

Reply:
I think it is reasonable that when monitor deployment exist, we can start migrate work.